admin_app/pages/about.py

```diff
@@ -0,0 +1,23 @@
+import streamlit as st
```

Comment:
thanks for adding! an you also add admin_app/README with instructions on how to iterate / update/ how the pages work (more developer focused)

Reply:
sure added